minitwit/src/main/java/WebApplication.java

```diff
@@ -387,6 +396,116 @@ public static void updateLatest(Request request) {
         return "";
     };
 
+    public static Route serveSimFllws = (Request request, Response response) -> {
+
+        //update_latest(request); //TODO: call
+
+        var db = new SQLite();
+        var conn = db.getConnection();
+
+        var who_id = getUserID(db, request.params(":username"));
+
+        if (who_id == 0) {
+            conn.close();
+            response.status(404);
+            return false;
+        }
+
+        Gson gson = new Gson();
+        Fllws fllws = gson.fromJson(request.body(), Fllws.class);
+
+
+        if (Objects.equals(request.requestMethod(), "POST") && !fllws.follow.isEmpty()) {
```

Comment:
As I understand it, the API will send `{"unfollow": "username"}` on unfollowing, in which case `fllws.follow` will be NULL and this line will throw.

Reply:
makes sense, added a null check beforehand